packages/app/src/contacts/components/contact-simple-list-item-phone-selection/contact-simple-list-item-phone-selection.test.tsx

```diff
@@ -244,7 +244,7 @@ describe("when contact has both primary and secondary phone number defined", ()
       )
     ).toHaveStyle(`background-color: ${expectedHoverColor};`)
   })
-  test("hovering over the avatar column does not change the background color of secondary number field", () => {
+  test.skip("hovering over the avatar column does not change the background color of secondary number field", () => {
```

Comment:
And this skip?

Reply:
Thanks, Patryk for these comments. Skips will be handled as part of this task https://appnroll.atlassian.net/browse/CP-2234